### PR TITLE
test: Use 8 Gig images also for the Atomics and grow the docker pool

### PR DESCRIPTION
**Pull request by mvollmer**
Overflowing the pool might cause weird behavior and we seem to be
getting close sometimes.

### CODE REVIEW

**Comment by mvollmer** [comment]
rhel-atomic is on its way.

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-atomic

**Comment by cockpituous** [comment]
Image creation for fedora-atomic in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-atomic-cc9668db2dd338421515c956a2ac33d94651e73f/

**Comment by cockpituous** [comment]
Image creation done: https://github.com/cockpituous/cockpit/commits/refresh-fedora-atomic-cc9668db2dd338421515c956a2ac33d94651e73f

**Comment by mvollmer** [comment]
rhel-atomic is already 10G, so we don't need to make it bigger.  But the change top atomic.setup still affects it, so let's recreate it as well.

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-atomic

**Comment by cockpituous** [comment]
Image creation for fedora-atomic in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-atomic-31aad8f00d254cc5acdbbef756bfb80993f52c0a/

**Comment by cockpituous** [comment]
Image creation failed.

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-atomic

**Comment by cockpituous** [comment]
Image creation for fedora-atomic in process on cockpit-8.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-atomic-31aad8f00d254cc5acdbbef756bfb80993f52c0a/

**Comment by cockpituous** [comment]
Image creation done: https://github.com/cockpituous/cockpit/commits/refresh-fedora-atomic-31aad8f00d254cc5acdbbef756bfb80993f52c0a

File: test/images/scripts/lib/atomic.setup [hunk 1]
```diff
@@ -18,16 +18,29 @@
...
+# Checkout the just upgraded os branch since we'll use it every time
+# we build a new tree.
+#
+# HACK - Find a better way to compute the ref.
+# https://lists.projectatomic.io/projectatomic-archives/atomic-devel/2016-July/msg00015.html
+
+remote=$(ostree remote list)
 ref=$(python -c 'import json; print json.load(open("/usr/share/rpm-ostree/treefile.json"))["ref"]')
-ostree checkout "$ref" /var/local-tree
+ostree checkout "$remote:$ref" /var/local-tree
```
Comment (stefwalter) [inline test/images/scripts/lib/atomic.setup]: This seems to break the OSTree test case. Or am I misunderstanding the test results.

**Comment by mvollmer** [comment]
Old test failures:

https://fedorapeople.org/groups/cockpit/logs/pull-4677-c1ddc871-verify-fedora-atomic/log.html
https://fedorapeople.org/groups/cockpit/logs/pull-4677-c1ddc871-verify-rhel-atomic/log.html

**Comment by petervo** [comment]
I believe the failure here is a regression in the newer versions of rpm-ostree. I've filed an issue here: https://github.com/projectatomic/rpm-ostree/issues/392. I guess we need to decide if we want to make a known issue or hold off on updating these images till it gets fixed.

**Comment by mvollmer** [comment]
> I believe the failure here is a regression in the newer versions of rpm-ostree. I've filed an issue here: projectatomic/rpm-ostree#392.

Great, thanks.

> I guess we need to decide if we want to make a known issue or hold off on updating these images till it gets fixed.

I make a known issue.

**Pull request closed without merging**